### PR TITLE
Feat/carousel branch

### LINKED ISSUE #1: As a user, when I visit the app, all movies should be displayed

### LINKED ISSUE #9: Improve accessibility through tabbing

### LINKED ISSUE #10: Center badges on mobile views

### PR DESCRIPTION
**Pull request by camjohnson-code**
# Pull Request

## Description
This PR adds in the functionality for Iteration 1. It adds in two different carousels for users to click through and it shows all the movies.

## Changes
- Added a carousel for popular movies
- Added a carousel for recommended movies
- Added a section that displays all the movies
- Added tabindex to the titles of each movie card so users can tab through it
- Added styles to all the components

## Related Issues
- Addresses Issue #9 
- Addresses Issue #1 
- Addresses Issue #10 

## Checklist
- [x] I have read the contribution guidelines.
- [x] I have tested the changes locally.
- [x] The code follows the project's coding standards.
- [x] Documentation has been updated (if applicable).
- [x] I have added/updated unit tests (if applicable).
- [x] I have squashed/organized my commits.

## Additional Notes
None.

### CODE REVIEW

**Comment by EGavinG** [comment]
Successfully completed Iteration 1 collaboratively and conducted a comprehensive review of meeting acceptance criteria end goals. Great job Cameron! Merging to Main.

File: src/Components/App/App.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import AllMovies from '../Movies Display/All-Movies';
+
+const App = () => {
+  const [movies, setMovies] = useState(movieData.movies);
+
+  const getRandomMovies = (movies) => {
+    const randomMovies = movies.slice();
+
+    for (let i = randomMovies.length - 1; i > 0; i--) {
+      const j = Math.floor(Math.random() * (i + 1));
```
Comment (nicktu12) [inline src/Components/App/App.js:16]: Nice randomizing!

File: src/Components/Carousel/Carousel.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      items: 1,
+    },
+  };
+
+  const isMobile = window.innerWidth <= 464;
+
+  return (
+    <div>
+      <h2 tabindex='0'>{badge} Movies</h2>
+      <Carousel
```
Comment (nicktu12) [inline src/Components/Carousel/Carousel.js:39]: So many props!! Third party libraries is like learning a whole new technology sometimes!

File: src/Components/Movies Display/All-Movies.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    },
+    mobile: {
+      breakpoint: { max: 464, min: 0 },
+      items: 1,
+    },
+  };
+
+  const isMobile = window.innerWidth <= 464;
+
+  return (<div responsive={responsive} deviceType={isMobile ? 'mobile' : 'desktop'}>
```
Comment (nicktu12) [inline src/Components/Movies Display/All-Movies.js:37]: the responsive attribute is pretty cool - make sure you know whats going on here!

**Pull request merged**